test/census_test.go

```diff
@@ -202,10 +203,11 @@ func TestCensus(t *testing.T) {
 	resp = doRequest("getRoot", nil)
 	qt.Assert(t, resp.Root, qt.DeepEquals, root)
 
+	// TODO WIP getSize method is not currently supported
```

Comment:
The `getSize` will be supported once https://github.com/vocdoni/vocdoni-node/issues/271 is done (soon)